src/software/SfM/main_ComputeStructureFromKnownPoses.cpp

```diff
@@ -343,8 +343,8 @@ int main(int argc, char **argv)
       (
         sfm_data,
         Optimize_Options(
-          cameras::Intrinsic_Parameter_Type::ADJUST_ALL,
-          Extrinsic_Parameter_Type::ADJUST_ALL,
+          cameras::Intrinsic_Parameter_Type::NONE,
```

Comment:
He the adjustment of all the parameter was made on purpose.
We can extend the fine control on which info will be refined by extending the command line.

Reply:
You are right. It will be better to adjust these parameter in the command line.